File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/adma/ADM103UgrdUserGrpInfo.java
package nta.med.data.model.ihis.adma;

import java.io.Serializable;

public class ADM103UgrdUserGrpInfo implements Serializable {
	private String userGroup;
	private String groupNm;
	public ADM103UgrdUserGrpInfo(String userGroup, String groupNm) {
		super();
		this.userGroup = userGroup;
		this.groupNm = groupNm;
	}
	public String getUserGroup() {
		return userGroup;
	}
	public void setUserGroup(String userGroup) {
		this.userGroup = userGroup;
	}
	public String getGroupNm() {
		return groupNm;
	}
	public void setGroupNm(String groupNm) {
		this.groupNm = groupNm;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Adma/ADM103UgrdUserGrpInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Adma
{
    [Serializable]
    public class ADM103UgrdUserGrpInfo
    {
        private String _userGroup;
        private String _groupNm;
        private String _rowState;

        public String UserGroup
        {
            get { return this._userGroup; }
            set { this._userGroup = value; }
        }

        public String GroupNm
        {
            get { return this._groupNm; }
            set { this._groupNm = value; }
        }

        public String RowState
        {
            get { return this._rowState; }
            set { this._rowState = value; }
        }

        public ADM103UgrdUserGrpInfo() { }

        public ADM103UgrdUserGrpInfo(String userGroup, String groupNm, String rowState)
        {
            this._userGroup = userGroup;
            this._groupNm = groupNm;
            this._rowState = rowState;
        }

    }
}